ride_share.rb

```diff
@@ -0,0 +1,272 @@
+########################################################
+# Step 1: Establish the layers
+
+# In this section of the file, as a series of comments,
+# create a list of the layers you identify.
+# Which layers are nested in each other?
+# Which layers of data "have" within it a different layer?
+# Which layers are "next" to each other?
+
+########################################################
+# Step 2: Assign a data structure to each layer
+
+# Copy your list from above, and in this section
+# determine what data structure each layer should have
+
+########################################################
+# Step 3: Make the data structure!
+
+# Setup the entire data structure:
+# based off of the notes you have above, create the
+# and manually write in data presented in rides.csv
+# You should be copying and pasting the literal data
+# into this data structure, such as "DR0004"
+# and "3rd Feb 2016" and "RD0022"
+
+rides = [
+    # Ride 0
+    {
+        driver_id: "DR0004", 
```

Comment:
I looks like you have VS Code set up to use 4 spaces for indentation. At Ada we use 2 spaces, and it's important for everyone to match that so that wen we start sharing code between computers with Git everything lines up. Please change your editor settings:

1. `cmd+,` for settings
1. Search for `tab size`
1. Change the value for `Editor: Tab Size` to 2.